specification/dsdl/serialization.tex

```diff
@@ -487,13 +492,15 @@ \subsubsection{Tagged unions}
     Suppose that the value of \verb|b| is 7.
```

Comment:
Venturing to fix https://github.com/UAVCAN/specification/issues/75 here.

<img width="609" alt="image" src="https://user-images.githubusercontent.com/3298404/87249781-ce4d3d00-c469-11ea-9265-c204bb39df4b.png">

The selected fragment also needs updating. It should be `uint8` instead of `uint2`.

Reply:
ack, sorry. I'll get this.